mavros/README.md

```diff
@@ -50,9 +50,18 @@ Note: ids from URL overrides ids given by system\_id & component\_id parameters.
 Coordinate frames
 -----------------
 
-MAVROS does translate Aerospace NED frames, used in FCUs to ROS ENU frames.
-Rules descrided in [issue #49][iss49].
+MAVROS does translate Aerospace NED frames, used in FCUs to ROS ENU frames and vice-versa.
+Rules are descrided as follows:
 
+| Transform:             |                      |
+|:----------------------:|:--------------------:|
+| Position (local frame) | `(x,y,z)`→`(y,x,-z)` |
+
+| Transform:                     | Euler angles:         | Quaternions:              |
+|:------------------------------:|:---------------------:|:-------------------------:|
+| Orientation (body-fixed frame) | `(r,p,y)`→`(r,-p,-y)` | `(w,x,y,z)`→`(x,-y,-z,w)` |
```

Comment:
Look at position table too. That note for whole diff section.
And that minuses not descriptive, better to say what we do rotation around roll.